public-api/db-cdr/generate-cdr/csv/achilles_analysis.csv

```diff
@@ -40,3 +40,5 @@ analysis_id,"analysis_name","stratum_1_name","stratum_2_name","stratum_3_name","
 3321,"Age at occurrence","survey_concept_id","question_concept_id","","","age_decile","question_path","column","percentage"
 3331,"Sex Assigned at Birth","survey_concept_id","question_concept_id","answer_concept_id","answer_value_string","gender_concept_id","question_path","column","percentage_counts_by_question_counts"
 3332,"Age at occurrence","survey_concept_id","question_concept_id","answer_concept_id","answer_value_string","age_decile","question_path","column","percentage_counts_by_question_counts"
+3400,"Survey Module Counts By Version","survey_concept_id","version_id","version_month",NULL,NULL,NULL,"",""
```

Comment:
Adding analyses rows for survey participant and question counts for each version.